src/beast/beast/http/src/beast_http_method.cpp

```diff
@@ -120,5 +120,6 @@ status_text (int status)
     return "Unknown HTTP status";
 }
 
-}
-}
+} // http
```

Comment:
(Not in this change set). If we replaced the switch statements in `status_text` and `to_string` with static `flat_set` (or a sorted `std::vector`) that contained the strings, and the functions returned a `const&` we could save an alloc. Finding the right case might be faster as well, though I'm not sure of that.